stdlib/public/Concurrency/GlobalExecutor.cpp

```diff
@@ -169,9 +196,7 @@ void swift::swift_task_enqueueGlobal(Job *job) {
 
   JobPriority priority = job->getPriority();
 
-  // TODO: cache this to avoid the extra call
-  auto queue = dispatch_get_global_queue((dispatch_qos_class_t) priority,
-                                         /*flags*/ 0);
+  auto queue = getGlobalQueue(priority);
```

Comment:
I think there's a few more spots where we get this queue, might want to check and change those in the same PR? I think at least the Task sleep impl uses it too.

Reply:
Good call. Looks like a new spot was added after I created my branch. I'll rebase and switch the new one too.